podcasts/SwitchCell.swift

```diff
@@ -46,4 +46,29 @@ class SwitchCell: ThemeableCell {
 
     override func setSelected(_ selected: Bool, animated: Bool) {}
     override func setHighlighted(_ highlighted: Bool, animated: Bool) {}
+
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        setupAccessibility()
+    }
+
+    override func accessibilityActivate() -> Bool {
+        cellSwitch.setOn(!cellSwitch.isOn, animated: true)
+        cellSwitch.sendActions(for: .valueChanged)
+        matchAccessibilityValueWithSwitchControl()
+        return true
+    }
+
+    private func setupAccessibility() {
+        isAccessibilityElement = true
+        let labelText = cellLabel.text ?? ""
+        let secondaryLabelText = cellSecondaryLabel.text ?? ""
+        accessibilityLabel = "\(labelText), \(secondaryLabelText)"
+        accessibilityTraits = cellSwitch.accessibilityTraits
```

Comment:
I tried manually setting .switch as accessibility trait, but such trait does not exist. Accessibility trait for switch is not exposed and this is the only way to match behaviour like in Apple's apps.

Reply:
To behave like Apple apps you only have to add the switch to the `accessoryView` instead of `contentView` of the cell.
I can't test on a real device right now but you can see in the video that the cell is read entirely and preforming the action (double tap) activates/deactivates the switch. 
Hope it helps.

https://user-images.githubusercontent.com/7861088/197657365-78441bb8-3385-4bb1-ac9d-7c565460ecad.mov

Reply:
> To behave like Apple apps you only have to add the switch to the `accessoryView` instead of `contentView` of the cell.
> 
> I can't test on a real device right now but you can see in the video that the cell is read entirely and preforming the action (double tap) activates/deactivates the switch. 
> 
> Removing the switch from interface builder, changing line 4 to `var cellSwitch = ThemeableSwitch(frame: .zero)` and adding it to cell's `accessoryView` ( `self.accessoryView = cellSwitch`) should fix the behavior.
> 
> Hope it helps.
> 
> 
> 
> https://user-images.githubusercontent.com/7861088/197657365-78441bb8-3385-4bb1-ac9d-7c565460ecad.mov
> 
> 
> 
> 

Thanks for the comment. I am aware of accessory view and accessibility behaviour when switch is its subview. I wanted to make as little changes as possible.

Reply:
Oh ok.
You can try to hide the switch from the voiceover and set a `accessibilityCustomActions` for the cell to change the switch value programmatically